src/scripts/main.js

```diff
@@ -354,7 +354,43 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+for (let person of people) {
+  const row = document.createElement('tr');
+
+  if (person.sex === 'f') {
+    person.sex = 'Female';
+  }
+
+  if (person.sex === 'm') {
+    person.sex = 'Male';
+  }
```

Comment:
Directly modifying the `sex` property of the `person` object can lead to unintended side effects, especially if the `people` array is used elsewhere in the code. Consider using a separate variable to store the display value for gender.